Command/PasswordResetRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\Exception\InterfaceNotImplemented;
+use SumoCoders\FrameworkMultiUserBundle\User\PasswordResetInterface;
+use SumoCoders\FrameworkMultiUserBundle\User\UserInterface;
+
+class PasswordResetRequest
```

Comment:
ok, dat klinkt logisch. Aangezien dit een command is zou ik het wel `RequestPasswordReset` noemen. Dat is de actie die de gebruiker doet. (commands beginnen normaal altijd met een werkwoord)